core/dbt/parser/manifest.py

```diff
@@ -322,7 +358,10 @@ def create_manifest(self) -> Manifest:
         )
         manifest.patch_nodes(self.results.patches)
         manifest.patch_macros(self.results.macro_patches)
```

Comment:
Should we toss the time it takes for patching nodes and macros?

Reply:
Toss the time? They're not being timed now. Not sure what you mean.

Reply:
whoops, I guess I meant "toss the time in". Since we are timing the other `patch_*` steps, should we include these as well?

Reply:
Those two things lines use a fraction of a second even in my big test suite, so there's not a need to time them.